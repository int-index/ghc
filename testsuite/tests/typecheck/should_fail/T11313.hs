{-# LANGUAGE TypeApplications #-}

module T11313 where

import Data.Kind

x = fmap @Type

-- test error message output, which was quite silly before
