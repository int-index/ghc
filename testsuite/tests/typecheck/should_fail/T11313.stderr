
T11313.hs:7:11: error:
    • Expected kind ‘* -> *’, but ‘Type’ has kind ‘*’
    • In the type ‘Type’
      In the expression: fmap @Type
      In an equation for ‘x’: x = fmap @Type
